lib/components/modals/account_modals.dart

```diff
@@ -771,7 +772,9 @@ class _AccountCreationConfirmContentState
                                   isEvmClaimed: false,
                                   iconSVG: widget.account!.svg,
                                 );
+                                if(hasThresholdBalance()){
```

Comment:
here https://github.com/reef-chain/mobile-app/blob/c580d3adfd869cfcf22bca009b23ae943023d187/lib/utils/bind_evm.dart#L27C26-L27C46
we are checking balances of only !isEvmClaimed
why? Any account is eligible to transfer to newly created account so it can bind.
Check if removeing this is affecting call in other places. Also refactor hasThresholdBalance() to anyAccountHasBalance(BigInt minBalance)